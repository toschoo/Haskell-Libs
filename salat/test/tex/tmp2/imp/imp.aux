\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\@writefile{toc}{\contentsline {chapter}{Contents}{iii}{section*.1}}
\@writefile{toc}{\contentsline {chapter}{Preface}{iv}{section*.3}}
\newmarginnote{note.4.1}{{4}{8332738sp}}
\newmarginnote{note.4.2}{{4}{28854073sp}}
\@writefile{toc}{\contentsline {chapter}{Editor's Note}{vi}{section*.5}}
\newmarginnote{note.6.1}{{6}{8332738sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {I}The Series of Natural Numbers}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.7.1}{{7}{8332738sp}}
\newmarginnote{note.7.2}{{7}{8717331sp}}
\newmarginnote{note.8.1}{{8}{8332738sp}}
\newlabel{change:pmyear}{{1}{2}{The Series of Natural Numbers\relax }{section*.7}{}}
\newmarginnote{note.9.1}{{9}{23636554sp}}
\newmarginnote{note.10.1}{{10}{12083776sp}}
\newmarginnote{note.10.2}{{10}{10657885sp}}
\newlabel{fn:example}{{2}{4}{The Series of Natural Numbers\relax }{section*.8}{}}
\newmarginnote{note.11.1}{{11}{28274284sp}}
\newmarginnote{note.12.1}{{12}{8332738sp}}
\newmarginnote{note.13.1}{{13}{16565127sp}}
\newmarginnote{note.14.1}{{14}{23324139sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {II}Definition of Number}{9}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.15.1}{{15}{8332738sp}}
\newmarginnote{note.15.2}{{15}{29255304sp}}
\newmarginnote{note.16.1}{{16}{21104156sp}}
\newmarginnote{note.17.1}{{17}{25344617sp}}
\newmarginnote{note.17.2}{{18}{27345089sp}}
\newmarginnote{note.18.1}{{18}{8332738sp}}
\newmarginnote{note.19.1}{{19}{28979534sp}}
\newlabel{fracchange17}{{II}{14}{Definition of Number\relax }{section*.9}{}}
\newmarginnote{note.19.2}{{20}{17520412sp}}
\newmarginnote{note.21.1}{{21}{26373977sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {III}Finitude and Mathematical Induction}{16}{chapter.3}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.22.1}{{22}{8332738sp}}
\newmarginnote{note.22.2}{{22}{27613699sp}}
\newlabel{change:notless}{{III}{17}{Finitude and Mathematical Induction\relax }{section*.10}{}}
\newmarginnote{note.23.1}{{23}{8332738sp}}
\newmarginnote{note.24.1}{{24}{8332738sp}}
\newmarginnote{note.25.1}{{25}{30276789sp}}
\newmarginnote{note.25.2}{{25}{30090358sp}}
\newmarginnote{note.26.1}{{26}{10988123sp}}
\newlabel{fracchange26}{{III}{21}{Finitude and Mathematical Induction\relax }{section*.11}{}}
\newmarginnote{note.27.1}{{27}{13654031sp}}
\newmarginnote{note.28.1}{{28}{16216679sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {IV}The Definition of Order}{23}{chapter.4}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.29.1}{{29}{8332738sp}}
\newmarginnote{note.29.2}{{29}{24587106sp}}
\newmarginnote{note.30.1}{{30}{23303669sp}}
\newmarginnote{note.31.1}{{31}{11422597sp}}
\newmarginnote{note.32.1}{{32}{10721194sp}}
\newmarginnote{note.32.2}{{32}{29754922sp}}
\newmarginnote{note.33.1}{{33}{23755293sp}}
\newmarginnote{note.34.1}{{34}{22266656sp}}
\newmarginnote{note.34.2}{{35}{12469746sp}}
\newmarginnote{note.35.1}{{36}{19899924sp}}
\newmarginnote{note.36.1}{{36}{21168144sp}}
\newmarginnote{note.37.1}{{37}{28186736sp}}
\newmarginnote{note.38.1}{{38}{22357197sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {V}Kinds of Relations}{33}{chapter.5}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.39.1}{{39}{8332738sp}}
\newmarginnote{note.39.2}{{39}{13007349sp}}
\newlabel{change:conversedomain}{{V}{33}{Kinds of Relations\relax }{section*.12}{}}
\newmarginnote{note.39.3}{{40}{11692572sp}}
\newmarginnote{note.40.1}{{41}{21340228sp}}
\newmarginnote{note.42.1}{{42}{27773008sp}}
\newmarginnote{note.42.2}{{42}{22833423sp}}
\newmarginnote{note.42.3}{{43}{25286207sp}}
\newmarginnote{note.42.4}{{44}{11613031sp}}
\newmarginnote{note.45.1}{{45}{29411760sp}}
\gdef \LT@i {\LT@entry 
    {1}{21.828pt}\LT@entry 
    {1}{21.828pt}\LT@entry 
    {1}{26.56801pt}}
\newmarginnote{note.46.1}{{46}{8332738sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {VI}Similarity of Relations}{41}{chapter.6}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.47.1}{{47}{8332738sp}}
\newmarginnote{note.47.2}{{47}{24711711sp}}
\newlabel{minor:page53}{{VI}{42}{Similarity of Relations\relax }{section*.13}{}}
\newlabel{change:minor53}{{VI}{42}{Similarity of Relations\relax }{section*.14}{}}
\newmarginnote{note.48.1}{{48}{22661954sp}}
\newmarginnote{note.49.1}{{49}{16356605sp}}
\newmarginnote{note.50.1}{{50}{21953485sp}}
\newlabel{minor:page56}{{VI}{44}{Similarity of Relations\relax }{section*.15}{}}
\newmarginnote{note.50.2}{{51}{13674200sp}}
\newmarginnote{note.51.1}{{51}{23196515sp}}
\newmarginnote{note.51.2}{{52}{14118276sp}}
\newmarginnote{note.51.3}{{53}{8744312sp}}
\newlabel{change:minor60}{{VI}{47}{Similarity of Relations\relax }{section*.16}{}}
\newmarginnote{note.54.1}{{54}{16329356sp}}
\newmarginnote{note.54.2}{{54}{17952207sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {VII}Rational, Real, and Complex Numbers}{50}{chapter.7}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.56.1}{{56}{8332738sp}}
\newmarginnote{note.56.2}{{56}{22217306sp}}
\newlabel{change:ornor}{{VII}{51}{Rational, Real, and Complex Numbers\relax }{section*.17}{}}
\newmarginnote{note.57.1}{{57}{22820515sp}}
\newmarginnote{note.58.1}{{58}{15818394sp}}
\newmarginnote{note.59.1}{{59}{23186127sp}}
\newmarginnote{note.59.2}{{59}{30634000sp}}
\newmarginnote{note.59.3}{{60}{29441456sp}}
\newmarginnote{note.61.1}{{61}{8332738sp}}
\newmarginnote{note.62.1}{{62}{12182954sp}}
\newmarginnote{note.62.2}{{63}{18352487sp}}
\newmarginnote{note.63.1}{{63}{10507888sp}}
\newmarginnote{note.64.1}{{64}{10833267sp}}
\newmarginnote{note.64.2}{{65}{29163591sp}}
\newmarginnote{note.65.1}{{66}{23260327sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {VIII}Infinite Cardinal Numbers}{62}{chapter.8}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.68.1}{{68}{8332738sp}}
\newmarginnote{note.68.2}{{68}{19535855sp}}
\newmarginnote{note.69.1}{{69}{13313974sp}}
\newmarginnote{note.70.1}{{70}{17280325sp}}
\newmarginnote{note.70.2}{{71}{16627198sp}}
\newmarginnote{note.71.1}{{71}{18518010sp}}
\newlabel{minor:progression}{{VIII}{66}{Infinite Cardinal Numbers\relax }{section*.18}{}}
\newmarginnote{note.72.1}{{72}{10709378sp}}
\newlabel{minor:progression2}{{VIII}{66}{Infinite Cardinal Numbers\relax }{section*.19}{}}
\newmarginnote{note.73.1}{{73}{25999996sp}}
\newlabel{page84display}{{VIII}{68}{Infinite Cardinal Numbers\relax }{section*.20}{}}
\newmarginnote{note.74.1}{{74}{25070091sp}}
\newlabel{change:deutschen}{{2}{68}{Infinite Cardinal Numbers\relax }{section*.21}{}}
\newmarginnote{note.74.2}{{75}{18522796sp}}
\newmarginnote{note.75.1}{{75}{17794477sp}}
\newmarginnote{note.76.1}{{76}{20334774sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {IX}Infinite Series and Ordinals}{72}{chapter.9}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.78.1}{{78}{8332738sp}}
\newmarginnote{note.78.2}{{78}{22274949sp}}
\newmarginnote{note.79.1}{{79}{29190880sp}}
\newlabel{style:times}{{IX}{73}{Infinite Series and Ordinals\relax }{section*.22}{}}
\newmarginnote{note.80.1}{{80}{8332738sp}}
\newmarginnote{note.81.1}{{81}{10252398sp}}
\newmarginnote{note.81.2}{{82}{13680266sp}}
\newmarginnote{note.82.1}{{82}{23106451sp}}
\newmarginnote{note.83.1}{{83}{20394442sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {X}Limits and Continuity}{78}{chapter.10}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.84.1}{{84}{8332738sp}}
\newmarginnote{note.84.2}{{84}{11815619sp}}
\newlabel{change:limitslimit}{{X}{79}{Limits and Continuity\relax }{section*.23}{}}
\newmarginnote{note.85.1}{{85}{31210171sp}}
\newlabel{parfraction1}{{X}{79}{Limits and Continuity\relax }{section*.24}{}}
\newmarginnote{note.86.1}{{86}{9607696sp}}
\newmarginnote{note.87.1}{{87}{17517186sp}}
\newmarginnote{note.87.2}{{88}{25094811sp}}
\newmarginnote{note.88.1}{{88}{10775375sp}}
\newmarginnote{note.89.1}{{89}{25889735sp}}
\newmarginnote{note.90.1}{{90}{20276695sp}}
\newmarginnote{note.90.2}{{91}{16074090sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XI}Limits and Continuity of Functions}{86}{chapter.11}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.92.1}{{92}{8332738sp}}
\newmarginnote{note.92.2}{{92}{8750014sp}}
\newmarginnote{note.93.1}{{93}{16220228sp}}
\newmarginnote{note.94.1}{{94}{22803386sp}}
\newlabel{change:valuevalues}{{XI}{88}{Limits and Continuity of Functions\relax }{section*.25}{}}
\newmarginnote{note.95.1}{{95}{22523447sp}}
\newmarginnote{note.95.2}{{95}{10758639sp}}
\newmarginnote{note.96.1}{{96}{23434027sp}}
\newlabel{change:advantage}{{XI}{91}{Limits and Continuity of Functions\relax }{section*.26}{}}
\newmarginnote{note.97.1}{{97}{10848811sp}}
\newmarginnote{note.98.1}{{98}{8771056sp}}
\newlabel{change:xz}{{XI}{92}{Limits and Continuity of Functions\relax }{section*.27}{}}
\newlabel{minor:115}{{XI}{92}{Limits and Continuity of Functions\relax }{section*.28}{}}
\newmarginnote{note.99.1}{{99}{11736819sp}}
\newlabel{parfraction2}{{XI}{93}{Limits and Continuity of Functions\relax }{section*.29}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XII}Selections\ and the Multiplicative Axiom}{94}{chapter.12}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.100.1}{{100}{8332738sp}}
\newmarginnote{note.100.2}{{100}{29021096sp}}
\newmarginnote{note.101.1}{{101}{14302747sp}}
\newmarginnote{note.102.1}{{102}{21981290sp}}
\newmarginnote{note.103.1}{{103}{8332738sp}}
\newlabel{change:minor121}{{XII}{97}{Selections\ and the Multiplicative Axiom\relax }{section*.30}{}}
\newmarginnote{note.104.1}{{104}{11397005sp}}
\newmarginnote{note.104.2}{{104}{29943769sp}}
\newmarginnote{note.105.1}{{105}{18584648sp}}
\newlabel{change:missingline}{{XII}{100}{Selections\ and the Multiplicative Axiom\relax }{section*.31}{}}
\newmarginnote{note.105.2}{{106}{15829801sp}}
\newmarginnote{note.107.1}{{107}{28485972sp}}
\newmarginnote{note.107.2}{{108}{23710191sp}}
\newmarginnote{note.108.1}{{108}{13811240sp}}
\newmarginnote{note.109.1}{{109}{24095729sp}}
\newlabel{change:ygammas}{{XII}{104}{Selections\ and the Multiplicative Axiom\relax }{section*.32}{}}
\newlabel{change:wellordered}{{XII}{104}{Selections\ and the Multiplicative Axiom\relax }{section*.33}{}}
\newmarginnote{note.109.2}{{110}{20151757sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XIII}The Axiom of Infinity and Logical Types}{106}{chapter.13}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.112.1}{{112}{8332738sp}}
\newlabel{change:minor131}{{XIII}{106}{The Axiom of Infinity and Logical Types\relax }{section*.34}{}}
\newmarginnote{note.112.2}{{112}{31480377sp}}
\newmarginnote{note.113.1}{{113}{16825495sp}}
\newmarginnote{note.114.1}{{114}{19600119sp}}
\newmarginnote{note.115.1}{{115}{8332738sp}}
\newmarginnote{note.115.2}{{116}{29223929sp}}
\newmarginnote{note.116.1}{{116}{24266293sp}}
\newlabel{change:minor137}{{XIII}{110}{The Axiom of Infinity and Logical Types\relax }{section*.35}{}}
\newmarginnote{note.117.1}{{117}{30274957sp}}
\newmarginnote{note.118.1}{{118}{26284126sp}}
\newlabel{change:socrates}{{XIII}{112}{The Axiom of Infinity and Logical Types\relax }{section*.36}{}}
\newmarginnote{note.118.2}{{119}{16738015sp}}
\newmarginnote{note.119.1}{{119}{27414381sp}}
\newmarginnote{note.120.1}{{120}{18308272sp}}
\newmarginnote{note.120.2}{{121}{25791451sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XIV}Incompatibility and the Theory of Deduction}{116}{chapter.14}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.122.1}{{122}{8332738sp}}
\newmarginnote{note.122.2}{{122}{16153664sp}}
\newmarginnote{note.123.1}{{123}{12615617sp}}
\newmarginnote{note.124.1}{{124}{10801797sp}}
\newmarginnote{note.125.1}{{125}{23843063sp}}
\newmarginnote{note.126.1}{{126}{10317500sp}}
\newmarginnote{note.126.2}{{126}{10641493sp}}
\newmarginnote{note.127.1}{{127}{8332738sp}}
\newmarginnote{note.128.1}{{128}{8332738sp}}
\newmarginnote{note.129.1}{{129}{29716961sp}}
\newmarginnote{note.129.2}{{130}{14621011sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XV}Propositional Functions}{125}{chapter.15}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.131.1}{{131}{8332738sp}}
\newmarginnote{note.131.2}{{131}{22185784sp}}
\newmarginnote{note.132.1}{{132}{27963046sp}}
\newmarginnote{note.132.2}{{133}{8699388sp}}
\newmarginnote{note.133.1}{{134}{30547531sp}}
\newlabel{change:minor159}{{XV}{128}{Propositional Functions\relax }{section*.37}{}}
\newmarginnote{note.134.1}{{134}{12545083sp}}
\newlabel{change:fnswapone}{{XV}{129}{Propositional Functions\relax }{section*.38}{}}
\newmarginnote{note.135.1}{{135}{8332738sp}}
\newlabel{change:pfeifferphi}{{XV}{129}{Propositional Functions\relax }{section*.39}{}}
\newlabel{change:fnswaptwo}{{XV}{129}{Propositional Functions\relax }{section*.40}{}}
\newmarginnote{note.135.2}{{136}{29667397sp}}
\newlabel{change:qm}{{XV}{131}{Propositional Functions\relax }{section*.41}{}}
\newmarginnote{note.137.1}{{137}{8332738sp}}
\newmarginnote{note.137.2}{{137}{28673838sp}}
\newmarginnote{note.138.1}{{138}{14415746sp}}
\newmarginnote{note.138.2}{{139}{30061888sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XVI}Descriptions}{134}{chapter.16}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.140.1}{{140}{8332738sp}}
\newmarginnote{note.140.2}{{140}{11373735sp}}
\newmarginnote{note.141.1}{{141}{14095407sp}}
\newmarginnote{note.141.2}{{142}{24918248sp}}
\newmarginnote{note.142.1}{{143}{9487798sp}}
\newmarginnote{note.143.1}{{143}{22503818sp}}
\newmarginnote{note.144.1}{{144}{15035716sp}}
\newlabel{change:manmen}{{XVI}{138}{Descriptions\relax }{section*.42}{}}
\newmarginnote{note.145.1}{{145}{28483740sp}}
\newmarginnote{note.145.2}{{146}{20836607sp}}
\newlabel{change:missingphi175}{{XVI}{140}{Descriptions\relax }{section*.43}{}}
\newmarginnote{note.145.3}{{147}{25247897sp}}
\newlabel{minor:waverley}{{XVI}{141}{Descriptions\relax }{section*.44}{}}
\newlabel{change:firstmissingphi}{{XVI}{141}{Descriptions\relax }{section*.45}{}}
\newmarginnote{note.145.4}{{147}{16665244sp}}
\newmarginnote{note.148.1}{{148}{22189605sp}}
\newmarginnote{note.148.2}{{149}{8698669sp}}
\newmarginnote{note.150.1}{{150}{14126510sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XVII}Classes}{145}{chapter.17}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.151.1}{{151}{8332738sp}}
\newmarginnote{note.151.2}{{151}{13862951sp}}
\newmarginnote{note.152.1}{{152}{11846651sp}}
\newmarginnote{note.153.1}{{153}{27536536sp}}
\newmarginnote{note.153.2}{{154}{17809885sp}}
\newlabel{change:minor185}{{XVII}{148}{Classes\relax }{section*.46}{}}
\newmarginnote{note.154.1}{{154}{24899805sp}}
\newlabel{change:secondmissingphi}{{XVII}{149}{Classes\relax }{section*.47}{}}
\newmarginnote{note.154.2}{{155}{11882999sp}}
\newmarginnote{note.156.1}{{156}{16601800sp}}
\newmarginnote{note.157.1}{{157}{27334769sp}}
\newlabel{change:frenchman}{{XVII}{151}{Classes\relax }{section*.48}{}}
\newmarginnote{note.157.2}{{158}{24154955sp}}
\newmarginnote{note.158.1}{{158}{25366295sp}}
\newlabel{change:tau}{{XVII}{153}{Classes\relax }{section*.49}{}}
\newmarginnote{note.159.1}{{159}{19270954sp}}
\newmarginnote{note.160.1}{{160}{12376535sp}}
\@writefile{toc}{\contentsline {chapter}{\numberline {XVIII}Mathematics and Logic}{156}{chapter.18}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newmarginnote{note.162.1}{{162}{8332738sp}}
\newmarginnote{note.162.2}{{162}{8332738sp}}
\newlabel{change:numbersanumber}{{XVIII}{157}{Mathematics and Logic\relax }{section*.50}{}}
\newmarginnote{note.163.1}{{163}{20030875sp}}
\newmarginnote{note.164.1}{{164}{8706413sp}}
\newlabel{minor:italicsocrates}{{XVIII}{158}{Mathematics and Logic\relax }{section*.51}{}}
\newlabel{change:betas}{{XVIII}{158}{Mathematics and Logic\relax }{section*.52}{}}
\newmarginnote{note.164.2}{{165}{21437996sp}}
\newmarginnote{note.165.1}{{166}{18822499sp}}
\newmarginnote{note.166.1}{{166}{13326754sp}}
\newlabel{change:wordwords}{{XVIII}{161}{Mathematics and Logic\relax }{section*.53}{}}
\newmarginnote{note.167.1}{{167}{18131488sp}}
\newmarginnote{note.167.2}{{168}{8332738sp}}
\newmarginnote{note.169.1}{{169}{29516010sp}}
\newmarginnote{note.169.2}{{169}{12667486sp}}
\newmarginnote{note.170.1}{{170}{14788521sp}}
\newlabel{change:minor205}{{XVIII}{164}{Mathematics and Logic\relax }{section*.54}{}}
\newmarginnote{note.170.2}{{171}{26731109sp}}
\@writefile{toc}{\contentsline {chapter}{Index}{166}{section*.55}}
\newmarginnote{note.172.1}{{172}{8332738sp}}
\newlabel{change:fregeindex}{{XVIII}{167}{Index\relax }{section*.57}{}}
\newmarginnote{note.173.1}{{173}{22733616sp}}
\newlabel{minor:208}{{XVIII}{168}{Index\relax }{section*.58}{}}
\@writefile{toc}{\contentsline {chapter}{Appendix: Changes to Online Edition}{170}{section*.59}}
