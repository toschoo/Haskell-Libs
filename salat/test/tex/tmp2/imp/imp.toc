\contentsline {chapter}{Contents}{iii}{section*.1}
\contentsline {chapter}{Preface}{iv}{section*.3}
\contentsline {chapter}{Editor's Note}{vi}{section*.5}
\contentsline {chapter}{\numberline {I}The Series of Natural Numbers}{1}{chapter.1}
\contentsline {chapter}{\numberline {II}Definition of Number}{9}{chapter.2}
\contentsline {chapter}{\numberline {III}Finitude and Mathematical Induction}{16}{chapter.3}
\contentsline {chapter}{\numberline {IV}The Definition of Order}{23}{chapter.4}
\contentsline {chapter}{\numberline {V}Kinds of Relations}{33}{chapter.5}
\contentsline {chapter}{\numberline {VI}Similarity of Relations}{41}{chapter.6}
\contentsline {chapter}{\numberline {VII}Rational, Real, and Complex Numbers}{50}{chapter.7}
\contentsline {chapter}{\numberline {VIII}Infinite Cardinal Numbers}{62}{chapter.8}
\contentsline {chapter}{\numberline {IX}Infinite Series and Ordinals}{72}{chapter.9}
\contentsline {chapter}{\numberline {X}Limits and Continuity}{78}{chapter.10}
\contentsline {chapter}{\numberline {XI}Limits and Continuity of Functions}{86}{chapter.11}
\contentsline {chapter}{\numberline {XII}Selections\ and the Multiplicative Axiom}{94}{chapter.12}
\contentsline {chapter}{\numberline {XIII}The Axiom of Infinity and Logical Types}{106}{chapter.13}
\contentsline {chapter}{\numberline {XIV}Incompatibility and the Theory of Deduction}{116}{chapter.14}
\contentsline {chapter}{\numberline {XV}Propositional Functions}{125}{chapter.15}
\contentsline {chapter}{\numberline {XVI}Descriptions}{134}{chapter.16}
\contentsline {chapter}{\numberline {XVII}Classes}{145}{chapter.17}
\contentsline {chapter}{\numberline {XVIII}Mathematics and Logic}{156}{chapter.18}
\contentsline {chapter}{Index}{166}{section*.55}
\contentsline {chapter}{Appendix: Changes to Online Edition}{170}{section*.59}
