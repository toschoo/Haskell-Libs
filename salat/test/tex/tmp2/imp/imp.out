\BOOKMARK [0][-]{section*.1}{Contents}{}
\BOOKMARK [0][-]{section*.3}{Preface}{}
\BOOKMARK [0][-]{section*.5}{Editor's Note}{}
\BOOKMARK [0][-]{chapter.1}{I The Series of Natural Numbers}{}
\BOOKMARK [0][-]{chapter.2}{II Definition of Number}{}
\BOOKMARK [0][-]{chapter.3}{III Finitude and Mathematical Induction}{}
\BOOKMARK [0][-]{chapter.4}{IV The Definition of Order}{}
\BOOKMARK [0][-]{chapter.5}{V Kinds of Relations}{}
\BOOKMARK [0][-]{chapter.6}{VI Similarity of Relations}{}
\BOOKMARK [0][-]{chapter.7}{VII Rational, Real, and Complex Numbers}{}
\BOOKMARK [0][-]{chapter.8}{VIII Infinite Cardinal Numbers}{}
\BOOKMARK [0][-]{chapter.9}{IX Infinite Series and Ordinals}{}
\BOOKMARK [0][-]{chapter.10}{X Limits and Continuity}{}
\BOOKMARK [0][-]{chapter.11}{XI Limits and Continuity of Functions}{}
\BOOKMARK [0][-]{chapter.12}{XII Selections and the Multiplicative Axiom}{}
\BOOKMARK [0][-]{chapter.13}{XIII The Axiom of Infinity and Logical Types}{}
\BOOKMARK [0][-]{chapter.14}{XIV Incompatibility and the Theory of Deduction}{}
\BOOKMARK [0][-]{chapter.15}{XV Propositional Functions}{}
\BOOKMARK [0][-]{chapter.16}{XVI Descriptions}{}
\BOOKMARK [0][-]{chapter.17}{XVII Classes}{}
\BOOKMARK [0][-]{chapter.18}{XVIII Mathematics and Logic}{}
\BOOKMARK [0][-]{section*.55}{Index}{}
\BOOKMARK [0][-]{section*.59}{Appendix: Changes to Online Edition}{}
